server/src/main/java/io/crate/breaker/TypedRowAccounting.java

```diff
@@ -35,6 +35,19 @@
  */
 public class TypedRowAccounting implements RowAccounting<Row> {
 
+    public static RowAccounting<Row> NO_ACCOUNTING = new RowAccounting<>() {
```

Comment:
Can you add this to the interface instead to match `RamAccounting.NO_ACCOUNTING`